autodepgraph/graph.py

```diff
@@ -5,13 +5,20 @@
 from autodepgraph.node import CalibrationNode
 import autodepgraph.visualization as vis
 import matplotlib.pyplot as plt
+import sys
 try:
     # Serves as a test to see if pyqtgraph is available
     import pyqtgraph as pg
     plot_mode = 'pg'
 except ImportError:
     plot_mode = 'mpl'
 
+try:
+    import pygraphviz
+except ImportError:
+    print('pygraphviz is not installed, plotting will be disabled', file=sys.stderr)
```

Comment:
I would use logging.warning for this. Is there any particular reason you used print to sys.stderr? If there is let me know and I'll probably merge.